internal/indexnode/param_table.go

```diff
@@ -58,8 +62,12 @@ func (pt *ParamTable) InitAlias(alias string) {
 func (pt *ParamTable) Init() {
 	once.Do(func() {
 		pt.BaseTable.Init()
+		if err := pt.LoadYaml("advanced/knowhere.yaml"); err != nil {
```

Comment:
I think the yaml file should be loaded in BaseTable.

Reply:
this config file can only be loaded into querynode and indexnode. so it's ok to be loaded in need.